src/initializers/bull/bull.ts

```diff
@@ -0,0 +1,121 @@
+import { defaultsDeep, has } from 'lodash';
+import * as cron from 'node-cron';
+import { getLogger } from '../log4js';
+import requireInjected from '../../require-injected';
+const Queue = requireInjected('bull');
+const newrelic = process.env.NEW_RELIC_LICENSE_KEY ? requireInjected('newrelic') : null;
+
+export default class Bull {
+  private logger = getLogger('orka.bull');
+  private prefix: string;
+  private defaultOptions: any;
+  private redisOpts: any;
+  private queueOpts: { [x: string]: { options: any } };
+  private instances = {};
+  private metricsTask: any;
+
+  constructor(prefix, queueOpts, defaultOptions, redisOpts) {
+    this.prefix = prefix;
+    this.defaultOptions = defaultOptions;
+    this.queueOpts = queueOpts.reduce((acc, q) => {
+      acc[q.name] = q;
+      return acc;
+    }, {});
```

Comment:
nit since you use lodash:

```suggestion
    this.queueOpts = keyBy(queueOpts, 'name');
```